### PR TITLE
Translate dev js fetch.md to zh

### PR DESCRIPTION
**Pull request by sunnywa**
Translate dev js fetch.md to zh
Signed-off-by: 王琦 <1099046071@qq.com>

### CODE REVIEW

File: docs/book/zh/src/dev/js/fetch.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# Fetch API
+# The `fetch` API
+
+`fetch` API 被广泛用于浏览器和基于 Node 的 JavaScript 应用程序中，用来通过网络获取内容。WasmEdge QuickJS 运行时在其非阻塞的 Aysnc 网络套接字 API 的基础上支持 `fetch`API。这使得很多 JS 的 API 和模块可以开箱即用。
+
+[example_js/wasi_http_fetch.js](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/wasi_http_fetch.js) 例子展示了如何在 WasmEdge 中使用`fetch` API。下面的代码片段显示了一个异步的 HTTP GET 请求。当程序等待和处理 GET 内容的时候，它可以开始另一个请求。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/fetch.md:5]: `fetch` 前后空一格

**Review by alabulei1 — COMMENTED** [review]
Thanks for your contribution. Please check out my feedback.

**Comment by sunnywa** [comment]
@alabulei1  fixed.

**Comment by alabulei1** [comment]
LGTM!

@hydai  Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Comment by hydai** [comment]
截圖 2022-02-16 at 17 38 51@2x
Looks like there are some conflicts which need to be resolved in the local side. @sunnywa Could you please check this. Thanks.

**Pull request merged**